Allow SYS_sched_getparam and SYS_sched_getscheduler
musl uses them for pthread_getschedparam()

source: https://git.alpinelinux.org/aports/commit/community/chromium?id=54af9f8ac24f52d382c5758e2445bf0206eff40e

--- services/service_manager/sandbox/linux/bpf_renderer_policy_linux.cc.orig	2019-10-08 21:03:18.253080425 +0200
+++ services/service_manager/sandbox/linux/bpf_renderer_policy_linux.cc	2019-10-08 21:04:19.648549718 +0200
@@ -88,10 +88,10 @@
     case __NR_sysinfo:
     case __NR_times:
     case __NR_uname:
-      return Allow();
-    case __NR_sched_getaffinity:
     case __NR_sched_getparam:
     case __NR_sched_getscheduler:
+      return Allow();
+    case __NR_sched_getaffinity:
     case __NR_sched_setscheduler:
       return sandbox::RestrictSchedTarget(GetPolicyPid(), sysno);
     case __NR_prlimit64:
